editor/src/components/editor/persistence/persistence.ts

```diff
@@ -133,7 +133,7 @@ export class PersistenceMachine {
 
     this.interpreter.start()
 
-    window.addEventListener('beforeunload', (e) => {
+    window.addEventListener('beforeunload', async (e) => {
```

Comment:
was this an eslint warning? the function doesn't seem to await anything

Reply:
Oh it did, I can remove that now.